recipe/gen_patch_json.py

```diff
@@ -1500,8 +1500,13 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             record["depends"] = deps
 
         # pillow 7.1.0 and 7.1.1 break napari viewer but this wasn't dealt with til latest release
-        if record_name == "napari" and record.get("timestamp", 0) < 1642529454000:  # 2022-01-18
-            _replace_pin("pillow", "pillow !=7.1.0,!=7.1.1", record.get("depends", []), record)
+        if record_name == "napari":
+            timestamp = record.get("timestamp", 0) 
+            if timestamp < 1642529454000:  # 2022-01-18
+                _replace_pin("pillow", "pillow !=7.1.0,!=7.1.1", record.get("depends", []), record)
+            if timestamp < 1661793597230:  # 2022-08-29
+                _replace_pin("vispy >=0.9.4", "vispy >=0.9.4<0.10", record.get("depends", []), record)
+                _replace_pin("vispy >=0.6.4", "vispy >=0.6.4<0.10", record.get("depends", []), record)
```

Comment:
Agreed. It should be...

```suggestion
                _replace_pin("vispy >=0.9.4", "vispy >=0.9.4,<0.10", record.get("depends", []), record)
                _replace_pin("vispy >=0.6.4", "vispy >=0.6.4,<0.10", record.get("depends", []), record)
```

Reply:
See PR ( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/333 )

Reply:
Just made a PR with that:
https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/333